src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -381,6 +389,37 @@ def build_cmd(self, env, host, **kwargs):
 
         return tst_cmd
 
+    def convert_xml(self, xml_file):
+        """Modify the xml file"""
+
+        fd = open(xml_file, 'r')
+        ofd = open('{}.xml'.format(xml_file), 'w')
+        for line in fd:
+            if self.src_dir in line:
+                L = re.sub('<dir>\/*' + self.src_dir + '\/*', '<dir>', line)
+                ofd.write(L)
+            else:
+                ofd.write(line)
+        os.unlink(xml_file)
+
+    def convert_xml_files(self):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.